.github/workflows/deployment.yml

```diff
@@ -1,4 +1,4 @@
-name: Deployment
+name: CD
```

Comment:
Two opinions here
- Calling this continuous deployment is maybe a bit much, we're not actually deploying to production from here - it's only part of the story. I think Deployment is actually accurate.
- I'd prefer not to use acronyms. (e.g. Continuous Deployment vs. Continuous Delivery)